translations/components.tex/cs.po

```diff
@@ -431,7 +431,7 @@ msgstr ""
 "      \\node at (0, 0) {\\includegraphics[width=0.6\\linewidth]{\\cards/mmback.png}};\n"
 "      \\node at (1.5, 0.1) {\\includegraphics[width=0.6\\linewidth]{\\cards/artifact-front.png}};\n"
 "    \\end{tikzpicture}\n"
-"    \\caption{\\textbf{\\pagelink{Artifact}{Artefakt}}}\n"
+"    \\caption{\\textbf{\\pagelink{Artifact}{Karta Artefaktu}}}\n"
```

Comment:
Some terms in the "Component" section were intentionally shortened to fit. It now causes layout to overflow to next pages in printable version

https://github.com/Heegu-sama/Homm3BG/actions/runs/11044893987/artifacts/1980256486

<img width="677" alt="Screenshot 2024-09-26 at 10 57 25" src="https://github.com/user-attachments/assets/d5b9304d-7981-46f0-903c-83f99fd506e2">

Reply:
I thought it might happen. I'll install TexStudio and somehow try to get po4a working on Windows so I can do change checks and make sure it doesn't happen again.
And while I'm looking at the attached picture, I forgot to change "Posílená Karta Statistik" to "Vylepšená Karta" in Components.

Reply:
The components section still has broken layout. What used to be on 2 pages is now on 4 with lots of empty space. I suggest to keep the original words and only update those addressed by this PR, keeping them short

Reply:
You can see tests below are failing because of that

Reply:
I agree. Please, could you edit that? I still can't get po4a to work on Windows to view the rulebook, and check exactly what all needs to be changed back.

Reply:
Im not sure how since your work on a fork.

@qwrtln Can we add @I3elphegor to maintainers here and recreate this PR here?

Reply:
I don't have such powers, but I can think of two easier solutions:
1. For @tomaas-zeman, since the following setting was ticked:
![image](https://github.com/user-attachments/assets/7aa6bf4d-40d6-4e43-bbc2-539ae4622130)
you can run `po4a`, and commit those changes yourself. See [instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork?platform=mac).
2. For @I3elphegor, you can open a PR in your fork, write a comment with text `po4a commit`, then the bot will run `po4a`, and commit changes to your PR. After that, you can merge the PR to the main branch in your fork, and those changes will be reflected here. See [example in another PR](https://github.com/Heegu-sama/Homm3BG/pull/442#issuecomment-2105990930), how this mechanism works.

Reply:
I fixed the page count by adjusting the size of some tokens and dice in the Components section.